Src/DependencyCollector/Microsoft.AI.DependencyCollector/project.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "2.3.0-beta2",
```

Comment:
Once these projects get migrated over to VS 2017, then we'll be able to reference a properties file for this reference. Until then, it needs to be hardcoded here.
I'll update it to 2.3.0-beta2.